sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -119,7 +119,8 @@ class BlobClient(StorageAccountHostsMixin, StorageEncryptionMixin):  # pylint: d
     :keyword str secondary_hostname:
         The hostname of the secondary endpoint.
     :keyword int max_block_size: The maximum chunk size for uploading a block blob in chunks.
```

Comment:
This isn't the right block size for append blobs. This the setting we use when chunking a blob for uploading a **block** blob and is just used internally (does not indicate the size of a block).

This would affect the `append_block` API in which it doesn't look like we document the block size. So, I think it's fine to revert these changes and just not have any comment updates. We tend to not document server-side limits anyway since they can change (as they did in this case).